### ISSUE TITLE
CardPlayer Card not loading

### ISSUE THREAD

**Issue by wirnico** [PaoloCuscela/Cards#94]
Thanks for your efforts to share with us these amazing Pods.
I encountered none problem with the CardPlayer Card. It is not loaded when view is loaded. Even in your demo project there is this issue.
If one clicks in the white area where the card should be it pops up and is then visible. Seems that the card infos are not being loaded.

Has someone a solution for me?

**Comment by tonycrencren** [comment]
Hello friend. I encountered the same issue. Did you ever get it to work?

**Comment by tonycrencren** [comment]
Hey. Thanks for posting this issue. It was a great reference for the stackoverflow question I just asked. All you have to do is add 

backgroundIV.frame.size.height = 300

To line 208 of CardPlayer.swift

https://stackoverflow.com/questions/54920415/cardplayer-card-not-loading/54922400?noredirect=1#comment96612189_54922400

**Comment by shadowsheep1** [comment]
Hi guys, as @tonycrencren pointed out, this is solved by adding an height to the frame on controller load. But I would suggest a more general way to do that like the one propose in the above SO answer:

```
// Issue is here
let currentHeight = backgroundIV.frame.size.height // <-- new line to store current value
/* this is line #207 */ backgroundIV.frame.size.height = originalFrame.height + ( isPresenting ? move/2 : 0 ) // This is line 207

// This is the check
if backgroundIV.frame.size.height <= 0 {
    print ("heigh is 0")
    backgroundIV.frame.size.height = currentHeight
}
```

First version of my answer maybe was not so clear so we ended up to try a more straight way by setting a fixed height like that

backgroundIV.frame.size.height = 300

Btw, @tonycrencren you did right posting a question on SO.

**Comment by tonycrencren** [comment]
nice. thanks again

**Comment by PaoloCuscela** [comment]
Thanks guys. Fixed in 1.4.0 ❤️
